### ISSUE TITLE
[SegmentedButton]: Segment sizes are independent of text length

### ISSUE THREAD

**Issue by sebkaim** [SAP/ui5-webcomponents#7955]
### Bug Description

In the SegmentedButton for instance, the individual segments take the length of the copy + padding according to the visual specs.

**[Visual Specs](https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2698912701)**
![image002](https://github.com/SAP/ui5-webcomponents/assets/152990669/82fd8565-936e-4f90-8c87-a9b09fd65b1d)

Yet, In the [implementation](https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-segmentedbutton--default&args=mode:SingleSelect;accessibleName:&globals=backgrounds.grid:!false), all button segments have the same size.

![image003](https://github.com/SAP/ui5-webcomponents/assets/152990669/f1803cb7-2a42-447e-887b-055a113706db)

### Affected Component

SegmentedButton

### Expected Behaviour

The individual segments (buttons) within a segmented button should take the length of their label, and not adopt the longest length fo all segments in a uniform way.

### Steps to Reproduce

See example images on top;

### Priority

Medium

### UI5 Web Components Version

1.5.0

### Browser

Chrome

### Operating System

Mac

### Organization

SAP Signavio

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-b could you check this, I think having the same size was something intentionally built.

**Comment by tsanislavgatev** [comment]
Yes, the default behaviour of the SegmentedButton is having equally sized items. But we've had discussions with designers and we are working on implementing a new, additional behaviour of the SegmentedButton where the Items will be fit-content.

**Comment by sebkaim** [comment]
[like]  Kaim, Sebastian reacted to your message:
________________________________

**Comment by texttechne** [comment]
Hi,
any news on this issue? 

I was a bit shocked to see equally sized buttons as default behavior: To me it looks like a bug; especially as the button borders are not visible. This only works [in the style guide](https://experience.sap.com/fiori-design-web/segmented-button-web-component/), because the used labels have kind of the same size (a typical use case of cheating designers 😉). I mean any kind of symmetry is betrayed by the following example (it looks sooo wrong that there is only this little padding for the last item) :

![Image](https://github.com/user-attachments/assets/d993aed7-1cfa-4201-a5bc-f9e73ef80085)

It also gives rise to issues regarding the available space. In this example the space is there for the full button text to be shown if the other buttons wouldn't take up that space:

![Image](https://github.com/user-attachments/assets/bf86a666-df64-438e-b99c-eeba818ac450)

**Comment by DMihaylova** [comment]
Hi @texttechne & @sebkaim,

Since version 2.16 there is an [itemsFitContent property](https://ui5.github.io/webcomponents/nightly/components/SegmentedButton/#itemsfitcontent). If you set it to true, you will achieve the requested behavior.

Regards,
Diana
